src/masternode/masternode-sync.h

```diff
@@ -45,18 +45,18 @@ class CMasternodeSync
     CMasternodeSync() { Reset(); }
 
 
-    void SendGovernanceSyncRequest(CNode* pnode, CConnman& connman);
+    static void SendGovernanceSyncRequest(CNode* pnode, CConnman& connman);
 
-    bool IsFailed() { return nCurrentAsset == MASTERNODE_SYNC_FAILED; }
-    bool IsBlockchainSynced() { return nCurrentAsset > MASTERNODE_SYNC_WAITING; }
-    bool IsSynced() { return nCurrentAsset == MASTERNODE_SYNC_FINISHED; }
+    bool IsFailed() const { return nCurrentAsset == MASTERNODE_SYNC_FAILED; }
+    bool IsBlockchainSynced() const { return nCurrentAsset > MASTERNODE_SYNC_WAITING; }
+    bool IsSynced() const { return nCurrentAsset == MASTERNODE_SYNC_FINISHED; }
 
-    int GetAssetID() { return nCurrentAsset; }
-    int GetAttempt() { return nTriedPeerCount; }
+    int GetAssetID() const { return nCurrentAsset; }
+    int GetAttempt() const { return nTriedPeerCount; }
     void BumpAssetLastTime(const std::string& strFuncName);
-    int64_t GetAssetStartTime() { return nTimeAssetSyncStarted; }
-    std::string GetAssetName();
-    std::string GetSyncStatus();
+    int64_t GetAssetStartTime() const { return nTimeAssetSyncStarted; }
+    std::string GetAssetName() const;
+    static std::string GetSyncStatus();
```

Comment:
Should instead s/`masternodeSync.nCurrentAsset`/`nCurrentAsset`/ in the implementation

Reply:
https://github.com/dashpay/dash/pull/3646/commits/6425f9c907463f69a577164e61dc7fd33268f188